pages/acquisition.md

```diff
@@ -10,11 +10,15 @@ link_cards:
     permalink: /implementation/
 ---
 
-There’s one problem that often blocks government from even getting started with the cloud: procurement. Getting a single cloud account can take six or more months.
+Acquisition is one of the biggest challenges agencies face when trying to get on the cloud. This is because cloud relies on consumption-based payment (i.e., pay only for what you use), while government relies on consistent, predictable billing models. The structures of government — some of which are required by law — often seem incompatible with the quick-moving pace of the private sector. This leads to frustration on the vendor side (as they’d like to move faster) and concern on the contracting side (as they need to make sure they’re following the rules). 
 
-Buying cloud is difficult because it's an entirely different business model than federal IT typically deals with. Usage-based pricing, intermediaries, commoditization and compliance can complicate and slow down the acquisition process.
+Here are some tips to keep in mind as you work with your contract officers. 
 
-Although there are hurdles to buying cloud, there are also strategies that can help. Understanding the issues at play can help you get through procurements successfully and efficiently, ensuring you get the products and services you need while saving taxpayer money.
+* **Understand the limits**. Current financial management systems weren’t designed for commercial cloud computing models; in fact, they work directly against the intended business advantages of cloud computing. If progress is slow, it isn’t because contracting officers are being contrary — it’s because they’re trying to fit a square peg into a round hole.
+* **Research different vehicles**. There are multiple ways to acquire cloud solutions, and each has its own strengths. [Within GSA alone](https://www.gsa.gov/technology/technology-programs) you can choose from IT Schedule 70, government-wide acquisition contracts (GWACs), and software blanket purchase agreements (BPAs). 
+* **Explore Acquisition Gateway**. [Acquisition Gateway](https://hallways.cap.gsa.gov/login-information) offers resources and contacts for both contract officers and buyers. Within the tool you can see current and past successful contracts, as well as points of contact to learn more.
+* **Share best practices**. The Cloud Center of Excellence (CCoE) has created the Cloud Adoption Survival Tips, Lessons Learned, and Experiences (CASTLE) guide, an extensive resource for acquisition professionals. Ask your contracting officers if they’ve seen it, and if not feel free to share.
```

Comment:
is there a link to this reference?

Reply:
@abbeykos updated the content since I posted so there is no longer a reference to CASTLE. I need to update the branch